src/lfn.c

```diff
@@ -85,8 +92,10 @@ WFFindFirst(
    lpFind->nSpaceLeft = MAXPATHLEN-nLen-1;
 
    if (lpFind->hFindFile != INVALID_HANDLE_VALUE) {
+      DWORD compareAttributes;
+      compareAttributes = lpFind->fd.dwFileAttributes & ATTR_USED;
       lpFind->dwAttrFilter = dwAttrFilter;

```

Comment:
This is not necessary as lpFind->fd.dwFileAttributes was already combined with ATTR_USED.

Reply:
lpFind->fd.dwFileAttributes are combined with ATTR_USED, then ATTR_JUNCTION/ATTR_SYMBOLIC are added (which are not part of ATTR_USED), then this compare occurs.  The intention was not to compare against ATTR_JUNCTION/ATTR_SYMBOLIC.  Another approach would be to perform this compare, and if the entry is filtered call FindNext (which adds them); if the entry is not filtered then add them here.

Reply:
I changed this to the alternate approach which avoids the back-and-forth conversion, effectively setting ATTR_JUNCTION/ATTR_SYMBOLIC after the decision to return the entry has already been made.